fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md

```diff
@@ -0,0 +1,204 @@
+
+1. Installer le Plugin sur tous les collecteurs Centreon supervisant des ressources Dell OS10 :
+
+```bash
+yum install centreon-plugin-Network-Dell-Os10-Snmp.noarch
+```
+
+2. Sur le serveur Central Centreon, installer le Plugin-Pack via le RPM:
+
+```bash
+yum install centreon-pack-network-dell-os10-snmp.noarch
+```
+
+3. Sur l'interface Web de Centreon, installer le Plugin-Pack *Dell OS10 SNMP* depuis la page "Configuration > Plugin packs > Manager"
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+Ce Plugin-Pack est conçu de manière à avoir dans Centreon un hôte par équipement Dell OS10.
+Lorsque vous ajoutez un hôte à Centreon, appliquez-lui le modèle Net-Dell-Os10-SNMP-custom. 
+Il est nécessaire de remplir les valeurs des champs "SNMP Community" et "SNMP Version".
+Une fois celui-ci configuré, certaines macros doivent être renseignées:
+
+:warning: Si vous utilisez SNMP version 3, sélectionnez la version SNMP appropriée 
+et configurez les paramètres SNMP v3 via la macro SNMPEXTRAOPTIONS.
+
+| Mandatory   | Name                    | Description                                                                                 |
+| :---------- | :---------------------- | :------------------------------------------------------------------------------------------ |
+|             | SNMPEXTRAOPTIONS        | Extra options SNMP de Dell Xseries                                                          |
+
+## FAQ
+
+#### Comment faire le test en ligne de commande et que signifient les principales options ?
+
+Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne de commande depuis votre collecteur Centreon avec l'utilisateur _centreon-engine_ :
+
+```bash
+/usr/lib/centreon/plugins/centreon_dell_os10_snmp.pl
+    --plugin=network::dell::os10::snmp::plugin
+    --mode=cpu
+    --hostname=10.30.2.114
+    --snmp-version='2c'
+    --snmp-community='dell_os10_ro'
+    --warning-average='60'
+    --critical-average='75'
+    --verbose
+	
+OK: CPU(s) average usage is 15.29 % - CPU '0' usage : 15.29 % 
+| 'total_cpu_avg'=15.29%;0:60;0:75;0;100 'cpu'=15.29%;;;0;100
+```
+
+Cette commande contrôle l'utilisation CPU (```--mode=cpu```) d'un équipement ayant pour adresse *10.30.2.114* (```--hostname=10.30.2.114```) 
+en version *2c* du protocol SNMP (```--snmp-version='2c'```) et avec la communauté *dell_os10_ro* (```--snmp-community='dell_os10_ro'```).
+
+Cette commande déclenchera une alarme WARNING si l'utilisation moyenne CPU est à plus de 60% (```--warning-average='60'```)
+et une alarme CRITICAL si plus de 75% (```--critical-average='75'```).
+
+Des seuils peuvent être fixés sur toutes les métriques de l'appareil en utilisant la syntaxe "```--warning-*metric* --critical-*metric*```".
+ 
+Toutes les options qui peuvent être utilisées avec ce plugin se trouvent sur la commande ```--help``` :
+
+```bash
+/usr/lib/centreon/plugins/centreon_dell_os10_snmp.pl --plugin=network::dell::os10::snmp::plugin \
```

Comment:
one option / line. Use \ between script name and --plugin

Reply:
Done ^^